### DISCUSSION TITLE
Can I specify the commit email in the github.dev web-based editor?

### DISCUSSION THREAD

**Discussion by wh201906** [community/community#33007] [Codespaces] [answered]
Can I specify the email address in the github.dev web-based editor when making a commit?

**Comment by GideonCheruiyot** [comment, marked as answer]
Hello @wh201906, github.dev does not support this currently, but you could change the email address to use by following these instructions: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-on-github.
